hack/testing/templates/logging-ci-test-runner-template.yaml

```diff
@@ -12,6 +12,8 @@ objects:
     namespace: ${TEST_NAMESPACE_NAME}
   spec:
     terminationGracePeriodSeconds: 0
+    nodeSelector:
```

Comment:
we need to restrict the test pod to a specific node?

Reply:
yes
there are a lot of implicit assumptions in our CI tests that there is only 1 fluentd running.  In order to make that work, we restrict fluentd to run only on the node where kibana is running (primarily so fluentd can see the container log files written by kibana).  Which means the test pod also has to run on the same node (so it can see the fluentd position files, log files, buffer files, etc.).